server/store/Scanning/cloudOCR.js

```diff
@@ -49,17 +48,23 @@ const transcriptParser = async paragraph => {
     words.push(word.symbols.map(symbol => symbol.text).join(''))
   );
 
-  const sentence = words.join(' ');
+  const removeCase = ['-', 'TOTAL', 'CUM', 'http', 'https'];
+
+  if (removeCase.some(el => words.includes(el))) return;
 
+  const sentence = words.join(' ');
   switch (true) {
     // Check for Enrolling Semester
-    case semesterStatus.some(status => sentence.includes(status)):
-      TranscriptMap.semesterList[currentSemIndex].status = 1;
+    case semesterStatus.some(status => sentence.includes(status)): {
+      const currentSem = TranscriptMap.semesterList[currentSemIndex];
+      currentSem.status = 1;
+      console.log(`[In Progress Sem]`, currentSem.term, currentSem.year);
```

Comment:
Now, I'll leave `console.log` for each case to make sure that it scans correctly.